addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14017,19 +14027,19 @@ msgstr ""
 #. Description of settings category "Appearance -> Skin" with label #166
 #: system/settings/settings.xml
 msgctxt "#36102"
-msgid "Category containing all Skin related settings."
+msgid "This category contains all skin related settings."
 msgstr ""
 
 #. Description of setting "Appearance -> Skin -> Skin" with label #166
 #: system/settings/settings.xml
 msgctxt "#36103"
-msgid "Select the skin for the user interface. This will define the look and feel of this application."
+msgid "Select the skin for the user interface. This will define the look and feel of the application."
 msgstr ""
 
 #. Description of setting "Appearance -> Skin -> Settings" with label #21417
 #: system/settings/settings.xml
 msgctxt "#36104"
-msgid "Change specific skin settings. The available options are dependent on the skin used."
+msgid "Change skin settings. The available options are specific to the skin used."
```

Comment:
IMO old phrasing is more correct. Maybe:
`Select to change skin specific settings. Which options are available for configuration depends on the features provided by the skin.`